gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -48,486 +90,660 @@ const paginationCodeExamples = [
+                the <NxCode>.nx-table-container</NxCode>'s content box for a paginated table with the
+                given parameters. The function, located in
+                the <NxCode>scss-shared/_nx-table-helpers.scss</NxCode> file, is named
+                <NxCode>pagination-table-height</NxCode>. See the description of its parameters below.
+              </NxP>
+              <NxTable>
+                <NxTable.Head>
+                  <NxTable.Row>
+                    <NxTable.Cell>Name</NxTable.Cell>
+                    <NxTable.Cell>Required</NxTable.Cell>
+                    <NxTable.Cell>Default Value</NxTable.Cell>
+                    <NxTable.Cell>Description</NxTable.Cell>
+                  </NxTable.Row>
+                </NxTable.Head>
+                <NxTable.Body>
+                  <NxTable.Row>
+                    <NxTable.Cell><NxCode>$body-row-count</NxCode></NxTable.Cell>
+                    <NxTable.Cell>Yes</NxTable.Cell>
+                    <NxTable.Cell>N/A</NxTable.Cell>
+                    <NxTable.Cell>
+                      The number of rows of content that the table body should have room for. This assumes that each
+                      row contains only a single line of text. Wrapping text or other elements that expand the size
+                      of any row will throw off the calculation.
+                    </NxTable.Cell>
+                  </NxTable.Row>
+                  <NxTable.Row>
+                    <NxTable.Cell><NxCode>$header-filter-row-count</NxCode></NxTable.Cell>
+                    <NxTable.Cell>No</NxTable.Cell>
+                    <NxTable.Cell>0</NxTable.Cell>
+                    <NxTable.Cell>
+                      The number of filter header rows on the table.
+                    </NxTable.Cell>
+                  </NxTable.Row>
+                  <NxTable.Row>
+                    <NxTable.Cell><NxCode>$header-row-count</NxCode></NxTable.Cell>
+                    <NxTable.Cell>No</NxTable.Cell>
+                    <NxTable.Cell>1</NxTable.Cell>
+                    <NxTable.Cell>
+                      The number of standard header rows on the table. This number should not include the count of any
+                      filter header rows.
+                    </NxTable.Cell>
+                  </NxTable.Row>
+                </NxTable.Body>
+              </NxTable>
+            </NxTile.Subsection>
+
+            <NxP>
+              For guidance on the construction of a scrolling table, see the scrolling example on
+              the <NxCode>nx-table-container</NxCode> HTML element page.
+            </NxP>
+          </GalleryDescriptionTile>
+        </NxTabPanel>
+
+        <NxTabPanel>
+
+          <GalleryExampleTile title="Simple Example"
+                              liveExample={NxTableSimpleExample}
+                              codeExamples={tableSimpleExampleCode}
+                              collapseCodeExample>
```

Comment:
If you get the latest from main I believe this prop is no longer needed/valid

Reply:
> If you get the latest from main I believe this prop is no longer needed/valid

https://github.com/sonatype/sonatype-react-shared-components/pull/607/commits/5a6b331a6becbaa1b60dc91b70bd8a059fa8f696